app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -122,6 +126,53 @@
         $node.attr("aria-expanded", state);
       }
 
+      function setSessionStorage() {
+        var isOpenSubsections = $('.subsection--is-open').length;
+        if (isOpenSubsections) {
+          var $openSubsections = $('.subsection--is-open');
+          $openSubsections.each(function(index) {
+            var subsectionOpenContentId = $(this).find('.subsection__content').attr('id');
+            sessionStorage.setItem( subsectionOpenContentId , subsectionOpenContentId);
```

Comment:
It looks like an example is a key of `subsection_content_1` with a value of `subsection_content_1`. And this session storage is shared across the www.gov.uk subdomain. Maybe a key of `service_manual_expand_topic_subsection_1` with a value of `true` will namespace it a little and make it a bit more explanatory if someone finds it and doesn't know what it is?